File: Hackerank/Days 6/main.js
const fs = require('fs');

function main() {
    const input = fs.readFileSync('/dev/stdin', 'utf-8');
    const lines = input.trim().split('\n');
    
    if (lines.length === 0 || lines[0].trim() === "") return;
    
    const t = parseInt(lines[0].trim(), 10);
    
    for (let i = 1; i <= t && i < lines.length; i++) {
        const palabra = lines[i].trim();
        let pares = "";
        let impares = "";
        
        for (let j = 0; j < palabra.length; j++) {
            if (j % 2 === 0) {
                pares += palabra[j];
            } else {
                impares += palabra[j];
            }
        }
        
        console.log(`${pares} ${impares}`);
    }
}

main();

File: Hackerank/Days 6/main.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
# Enter your code here. Read input from STDIN. Print output to STDOUT

t = int(input())

for _ in range(t):
    palabra = input()

    pares = ""
    impares = ""

    for i in range(len(palabra)):
        if i % 2 == 0:
            pares += palabra[i]
        else:
            impares += palabra[i]

    print(pares, impares)
